DB/index.js

```diff
@@ -1,40 +1,76 @@
-const mongoose = require('mongoose');
-const mongoURL = 'mongodb://localhost/carousel';
+// const mongoose = require('mongoose');
+// const mongoURL = 'mongodb://localhost/carousel';
 
 
-mongoose.connect(mongoURL, { useUnifiedTopology: true, useNewUrlParser: true })
-.then(()=>{
-  console.log(`connection to database established`)
-}).catch(err=>{
- throw err;
-})
-;
-const db = mongoose.connection;
+// mongoose.connect(mongoURL, { useUnifiedTopology: true, useNewUrlParser: true })
+// .then(()=>{
+//   console.log(`connection to database established`)
+// }).catch(err=>{
+//  throw err;
+// })
+// ;
+// const db = mongoose.connection;
+
 
 
+// module.exports = db;
 
-module.exports = db;
 
+const info = require('./environment.js');

```

Comment:
Can try out using a .env file and package `dotenv` to get the env variables. Should help you eliminate lines 21 to 25.